service/apiserver/internal/logic/transaction/sendtxlogic.go

```diff
@@ -28,8 +28,11 @@ func NewSendTxLogic(ctx context.Context, svcCtx *svc.ServiceContext) *SendTxLogi
 
 func (s *SendTxLogic) SendTx(req *types.ReqSendTx) (resp *types.TxHash, err error) {
 	resp = &types.TxHash{}
-	bc := core.NewBlockChainForDryRun(s.svcCtx.AccountModel, s.svcCtx.LiquidityModel, s.svcCtx.NftModel, s.svcCtx.TxPoolModel,
+	bc, err := core.NewBlockChainForDryRun(s.svcCtx.AccountModel, s.svcCtx.LiquidityModel, s.svcCtx.NftModel, s.svcCtx.TxPoolModel,
 		s.svcCtx.AssetModel, s.svcCtx.SysConfigModel, s.svcCtx.RedisCache)
+	if err != nil {
+		return nil, err
```

Comment:
do we need to wrap the error here, @forcodedancing

Reply:
Yes, it is better. Currently, we response `internal error`  to users. If we define a new error code, we can handle it better.